generic/utils/ras.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
```

Comment:
Please include the license heading here. You can inspire from other tests, eg: https://github.com/avocado-framework/avocado-misc-tests/blob/master/cpu/ebizzy.py

I'm not sure how this test evolved, so if it only existed in bash scripts, then you are most probably the original author of this script. If it already existed elsewhere, please include the `Based on ...` including link if it was public.

I found @lmr in one of the bash scripts, the rest is without heading. Ideally it'd be the best if you could add the header to all of the script files along with the original author and the license he chose. Keep in mind we only can include GPLv2+ compatible licensed files.